typescript-intro/typescript-intro-quiz-notes.md

```diff
@@ -0,0 +1,53 @@
+# typescript-intro-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- When was TypeScript introduced?
+
+October 2012
+
+- Who created TypeScript?
+
+Microsoft
+
+- What does the TypeScript `Compiler` do?
+
+Converts code from languages such as javascript or C++ into a code the computer can run.
```

Comment:
TS compiler takes your typescript code and converts into JS code so that your browser can run it. It also checks your code for type errors at compile time.